src/Microsoft.PowerShell.Security/security/CertificateProvider.cs

```diff
@@ -1,3 +1,4 @@
+#if !UNIX
```

Comment:
It'd be nice to do this within the project.json, but it looks like project.json does not support conditional inclusion and exclusion